applications/data_analysis/humid_heat_metrics.ipynb

```diff
@@ -0,0 +1,406 @@
+    "\n",
+    "{math}`WBGT_{od} = 0.7*T_{nwb} + 0.2*T_g + 0.1*T_a`\n",
+    "\n",
+    "{math}`WBGT_{id} = 0.7*T_{nwb} + 0.3*T_g`\n",
+    "\n",
+    "where {math}`T_a` refers to Dry Bulb Ambient Temperatue, {math}`T_{nwb}` is the Natural Wet Bulb Temperature with exposure to wind and sun, and {math}`T_g` is the Globe Temperature taken from inside a copper globe painted black and exposed to the sun{footcite}`li_2020`.\n",
+    "\n",
+    "However, this formula is complicated by the reality that Natural Wet Bulb Temperature and Globe Temperature are not always readily available variables from weather stations or atmospheric models.\n",
+    "\n",
+    "In this notebook, we will demonstrate the Australian Bureau of Meteorology (ABM) and Bernard methods of predicting wet bulb global temperature with a focus on the July 1995 Chicago heatwave.\n",
+    "\n",
+    "For our analysis, we have ERA5 reanalysis data for the lower contiguous United States (50&deg;N, 24&deg;S, -66&deg;E, -125&deg;W) from July 1995 with the variables: 2-meter temperature, 2-meter dew point temperature, surface pressure, and *u*/*v* wind components.\n",
+    "\n",
+    "ERA5 is a reanalysis global weather/climate dataset that combines model output and ovservationss with physical understanding to create a spatially and temporally consistent historic dataset, spanning from 1940 to today (updated every 5 days){footcite}`era5`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geocat.datafiles as gdf\n",
+    "import xarray as xr\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import cartopy.crs as ccrs\n",
+    "from scipy import optimize"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "era5 = xr.open_dataset(gdf.get(\"netcdf_files/era5_1995-07-14T12.nc\"))\n",
+    "era5.u10"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Convert from Kelvin to Celsius\n",
+    "\n",
+    "era5['t2m_C'] = era5['t2m'] - 273.15\n",
+    "era5['d2m_C'] = era5['d2m'] - 273.15"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Chicago coordinates\n",
+    "lat_chicago = 41.8781\n",
+    "lon_chicago = -87.6298"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Australian Bureau of Meteorology (ABM)\n",
+    "\n",
+    "The [Australian Bureau of Meteorology](http://www.bom.gov.au/)'s method of estimating Wet Bulb Global Temperature is attractive due to its simplicity. It only requires temperature and water pressure (or relative humidity){footcite}`abm_2010`.\n",
```

Comment:
```suggestion
    "The [Australian Bureau of Meteorology](http://www.bom.gov.au/)'s method of estimating Wet Bulb Global Temperature is attractive due to its simplicity. It only requires temperature and relative humidity {footcite}`abm_2010`.\n",
```

Water pressure is something a little different (esp. in hydrology / hydraulics).  I think just leaving it at RH works.

Reply:
Do you know why it says "this suggestion is out of date and cannot be applied? I'll do a work around, but I would like to just commit your change.

Reply:
I thought I changed it to "vapor pressure" that's probably why it is out of sync?

Reply:
Oh, maybe I found an older version?  Regardless, I'll go ahead and resolve these.